packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -70,6 +69,21 @@ - (UIViewController *)viewControllerWithWindow:(UIWindow *)window {
   return topController;
 }
 
+/**
+ * Applies UIImagePickerControllerCameraDevice on the UIImagePickerController.
```

Comment:
It doesn't though; it returns the device that should be used. How about "Returns the UIImagePickerControllerCameraDevice to use given [arguments]."